%Authors guidlines: http://royalsocietypublishing.org/instructions-authors
% 2500 words max (includes the title page, abstract, references, acknowledgements and figure/table legends)
% current version is around 3700. I think a big cut down can be done on the references.
% We allow a maximum of 4 displays, only 2 of which can be figures.

\documentclass[12pt,letterpaper]{article}


%Packages
\usepackage{pdflscape}
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\usepackage{longtable}
\pagenumbering{arabic}

\newcolumntype{L}[1]{>{\raggedright\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}

%Pagination style and stuff % NC: Note that these are all syst biol specific.
\linespread{2}
\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}
%\bibpunct{(}{)}{;}{a}{}{,}

%---------------------------------------------
%
%       START
%
%---------------------------------------------

\begin{document}

%Running head
\begin{flushright}
Version dated: \today
\end{flushright}

\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Assessment of available anatomical characters for phylogenetic analysis among living mammals}%Assessment of cladistic data availability for living mammals

\bigskip

\noindent {\normalsize \sc Thomas Guillerme$^1$$^,$$^*$ and Natalie Cooper$^1$$^,$$^2$}\\
\noindent {\small \it 
$^1$School of Natural Sciences, Trinity College Dublin, Dublin 2, Ireland.\\
$^2$Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7 5BD, UK.}\\
\end{center}
\medskip
\noindent{*\bf Corresponding author.} \textit{t.guillerme@imperial.ac.uk}\\  
\vspace{1in}

%Line numbering
\modulolinenumbers[1]
\linenumbers

%---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------
\newpage
\begin{abstract}
%\section{abstract} % NC: Just put this here as it then separates this for the word count
Analyses of living and fossil taxa are crucial for understanding changes in biodiversity through time.
The Total Evidence method allows living and fossil taxa to be combined in phylogenies, by using molecular data for living taxa and morphological data for both living and fossil taxa.
With this method, substantial overlap of anatomical characters among living and fossil taxa is crucial for accurately inferring topology.
However, although molecular data for living species is widely available, scientists using and generating morphological data mainly focus on fossils.
Therefore, there is a gap in our knowledge of anatomical characters available for living taxa even in well-studied groups such as mammals.

We investigated the amount of available anatomical characters for living mammals and how this data was phylogenetically distributed across orders.
22 of 28 mammalian orders have \textless 25\% species with available anatomical characters; this has implications for the accurate placement of fossil taxa, although the issue is less pronounced at higher taxonomic levels. 
In most orders, species with available data are randomly distributed across the phylogeny, which may reduce the impact of the problem.
We suggest that increased morphological data collection efforts for living taxa are needed to produce accurate Total Evidence phylogenies. 
\end{abstract}

\noindent Key words: Total Evidence method, phylogenetic clustering, cladistic matrix, extinct, topology\\
% NC: Might need some new ones as these are in the abstract now...

\vspace{1.5in}

%---------------------------------------------
%
%       INTRODUCTION
%
%---------------------------------------------
\newpage 
\section{Introduction}

There is an increasing consensus among biologists that studying both living and fossil taxa is essential for fully understanding macroevolutionary patterns and processes \cite{slaterunifying2013,fritzdiversity2013}.
To perform such analyses it is necessary to combine living and fossil taxa in phylogenetic trees.
One increasingly popular method, the Total Evidence method \cite{ronquista2012}, combines molecular data from living taxa and morphological data %TG: I left "morphological data" here since it TEM it doesn't have to be only anatomical characters
 from both living and fossil taxa in a supermatrix (e.g. \cite{pyrondivergence2011,ronquista2012,schragocombining2013,slaterunifying2013,beckancient2014}), producing a phylogeny with living and fossil taxa at the tips. 
A downside of this method is that it requires molecular data for living taxa and morphological data for both living and fossil taxa. %TG: same comment as above
Sections of this data can be difficult, or impossible, to collect for every taxon in the analysis.
For example, fossils rarely have molecular data and incomplete fossil preservation may restrict the amount of anatomical characters available.
Additionally, it has become less common to collect anatomical characters for living taxa when molecular data is available (e.g. in \cite{slaterphylogenetic2013}, only 13\% of living taxa have coded anatomical characters).
Unfortunately this missing data can lead to errors in phylogenetic inference.
Simulations show that the ability of the Total Evidence method to recover the correct topology decreases when there is little overlap between anatomical characters in living and fossil taxa, and that the effect of missing data on topology is greatest when living taxa have few anatomical characters available \cite{GuillermeCooper}.
This is because (1) fossils will not be placed accurately within the correct clade if it contains no anatomical characters for living taxa; and (2) fossils have a higher probability of branching within clades with more anatomical characters available for living taxa, regardless of whether this is the correct clade \cite{GuillermeCooper}. 

The issues above highlight that it is crucial to have sufficient anatomical characters available for living taxa in a clade before using a Total Evidence approach.
However, it is unclear how much anatomical characters are actually available for living taxa, i.e. already coded from museum specimens and deposited in phylogenetic matrices accessible online, and how this data is distributed across clades. % TG: emphasize the "availibility" definition her?
Intuitively, most people assume this kind of data has already been collected, but empirical data suggest otherwise (e.g. in \cite{ronquista2012,slaterphylogenetic2013,beckancient2014}).
To investigate this further, we assess the amount of available anatomical characters for living mammals to determine whether sufficient data exists to build reliable Total Evidence phylogenies in this group.
We also determine whether the available anatomical characters are phylogenetically overdispersed or clustered across mammalian orders.

%---------------------------------------------
%
%       METHODS
%
%---------------------------------------------
\section{Materials and Methods}
\subsection{Data collection and standardisation}
We downloaded all cladistic matrices %TG: I left "cladistic" here 'cause that's what the matrix is (even following Peter's definition)!
 containing any living and/or fossil mammal taxa from three major public databases: MorphoBank (\url{morphobank.org} \cite{morphobank}), Graeme Lloyd's website (\url{graemetlloyd.com/matrmamm.html}) and Ross Mounce's GitHub repository (\url{github.com/rossmounce/cladistic-data}).
We also performed a systematic Google Scholar search for matrices that were not uploaded to these databases (see Electronic Supplementary Material (ESM) for a detailed description of the search procedure).
In total, we downloaded 286 matrices containing 5228 unique operational taxonomic units (OTUs). 
We used OTUs rather than species because entries in the matrices ranged from species to families, and standardised the taxonomy as described in the ESM. 
We designated as ``living'' all OTUs that were either present in the phylogeny of \cite{BinindaEmonds} or the taxonomy of \cite{wilson2005mammal}.

% Matrices with few characters are problematic when comparing available data among matrices because (1) they have less chance of having overlapping characters with other matrices \cite{wagner2000} and (2) they are more likely to contain specific characters that are not applicable% rev 1.2.4
%  across large clades (e.g. ``antler ramifications'' is a character that is only applicable to Cervidae \cite{Brazeau2011}).
% Therefore we selected only matrices containing \textgreater 100 characters for each OTU.
% This threshold was chosen to correspond with the number of characters used in \cite{GuillermeCooper} and \cite{harrisonamong-character2014}.
% Results of analyses with no threshold are available in the ESM. 
% After removing matrices with \textless 100 characters, we retained 1074 unique living OTUs from 126 matrices. % 1601 unique living OTUs for 286 matrices (no threshold)
We retained 1601 unique living OTUs from 286 matrices.

\subsection{Data availability and distribution}
To assess the availability of anatomical characters for each mammalian order, we calculated the percentage of OTUs with cladistic data at three different taxonomic levels: family, genus and species. %TG: changed cladistic datat to anatomical characters
We consider orders with \textless 25\% of living taxa with available anatomical characters as having low data coverage, and orders with \textgreater 75\% of living taxa with available anatomical characters as having high data coverage. 

For each order, we investigated whether the available anatomical characters were (i) randomly distributed, (ii) overdispersed or (iii) clustered, with respect to phylogeny, using two metrics from community phylogenetics: the Nearest Taxon Index (NTI; \cite{webb2002phylogenies}) and the Net Relatedness Index (NRI; \cite{webb2002phylogenies}). 
NTI is most sensitive to clustering or overdispersion near the tips, whereas NRI is more sensitive to it across the whole phylogeny \cite{Cooper2008}. 
Both metrics were calculated using the \texttt{picante} package in R \cite{picante,R}.

NTI \cite{webb2002phylogenies} is based on mean nearest neighbour distance ($MNND$) and is calculated as follows:
  \begin{equation}
    \text{NTI}=-\left(\frac{\overline{MNND}_{obs}-\overline{MNND}_{n}}{\sigma(MNND_{n})}\right)
  \end{equation}
where $\overline{MNND}_{obs}$ is the observed mean sum of the branch lengths between each of $n$ taxa with available anatomical characters and its nearest neighbour with available anatomical characters in the phylogeny, 
$\overline{MNND}_{n}$ is the mean of 1000 $MNND$ between $n$ randomly drawn taxa, and $\sigma(MNND_{n})$ is the standard deviation of these 1000 random $MNND$ values.
NRI is calculated in the same way, but using the mean phylogenetic distance ($MPD$):
  \begin{equation}
    \text{NRI}=-\left(\frac{\overline{MPD}_{obs}-\overline{MPD}_{n}}{\sigma(MPD_{n})}\right)
  \end{equation}
where $\overline{MPD}_{obs}$ is the observed mean phylogenetic branch length of the tree containing only the $n$ taxa with available anatomical characters.
Negative NTI and NRI values show that the focal taxa are more overdispersed across the phylogeny than expected by chance, and positive values reflect clustering.

We calculated NTI and NRI values for each mammalian order separately, at each different taxonomic level. 
For each analysis our focal taxa were those with available anatomical characters at that taxonomic level and the phylogeny was that of the order pruned from \cite{BinindaEmonds}.

%---------------------------------------------
%
%       RESULTS
%
%---------------------------------------------


\section{Results}
11 out of 28 orders have low coverage (\textless 25\% species with available anatomical characters) and 7 have high coverage (\textgreater 75\% species with available anatomical characters) at the species-level.
At the genus-level, one order have low coverage and 15 have high coverage, and at the family-level, no orders have low coverage and 25 have high coverage (Table\ref{Table_results}).

%TG: former results:
% 22 of 28 orders have low coverage (\textless 25\% species with available anatomical characters) and six have high coverage (\textgreater 75\% species with available anatomical characters) at the species-level.
% At the genus-level, three orders have low coverage and 12 have high coverage, and at the family-level, no orders have low coverage and 23 have high coverage (Table\ref{Table_results}).

%TG: don't clean up this caption please!
%\caption{Number of taxa with available cladistic data for mammalian orders at three taxonomic levels. The left vertical bar represents low coverage (\textless 25\%); the right vertical bar represents high coverage (\textgreater 75\%). Negative Net Relatedness Index (NRI) and Nearest Taxon Index (NTI) values indicate phylogenetic overdispersion; positive values indicate phylogenetic clustering. Significant NRI or NTI values are in bold. *p \textless 0.05; **p \textless 0.01; ***p \textless 0.001.} \\ 
%TG: don't clean up this caption please!

\input{Table_results}

Only seven orders had significantly clustered data (Cetartiodactyla, Cingulata, Pilosa and Rodentia at the species-level and Carnivora, Chiroptera and Soricomorpha at both species- and genus-level) and none had significantly overdispersed data (Table~\ref{Table_results}).

Figure \ref{Figure_example_coverage} shows randomly distributed OTUs with cladistic data in Primates (Figure \ref{Figure_example_coverage}A) and phylogenetically clustered OTUs with cladistic data in Carnivora (mainly Canidae and Urisdae but no Herpestidae; Figure \ref{Figure_example_coverage}B).

\begin{figure}[!htbp]
\centering
    \includegraphics[width=1\textwidth]{figure1.pdf}
\caption{Phylogenetic distribution of species with available anatomical characters across two orders (A: Primates; B: Carnivora).
Blue branches indicate available anatomical characters for the species.}
\label{Figure_example_coverage}
\end{figure}

%---------------------------------------------
%
%       DISCUSSION
%
%---------------------------------------------

\section{Discussion}
Our results show that although phylogenetic relationships among living mammals are well-resolved (e.g. \cite{BinindaEmonds,meredithimpacts2011}), most of the data used to build these phylogenies is molecular, and few anatomical characters are available for living mammals compared to fossils (e.g. \cite{O'Leary08022013,ni2013oldest}).
This has implications for building Total Evidence phylogenies containing both living and fossil mammals, as without sufficient available anatomical characters for living species, fossil placements in these trees are very uncertain \cite{GuillermeCooper}.

The number of living mammalian OTUs with no available anatomical characters was surprisingly high at the species-level: only seven out of 28 orders have a high coverage of taxa with available anatomical characters.
This high coverage threshold of 75\% of taxa with available anatomical characters represents the minimum amount of data required before missing data has a significant effect on the topology of Total Evidence trees \cite{GuillermeCooper}.
Beyond this threshold, there is considerable displacement of wildcard taxa and decreased clade conservation \cite{GuillermeCooper}.
Therefore we expect difficulties in placement of fossils at the species-level in most mammalian orders, but fewer issues at higher taxonomic levels.
This point is important from a practical point of view because of the slight discrepancy between neontological and palaeontological species concepts.
While neontological species are described using morphology, genes, distribution etc.; palaeontological species can be based only on morphological, spatial and temporal data (e.g. \cite{ni2013oldest}).
Therefore, many palaeontological studies use the genus (or monospecific genera) as their smallest OTU (e.g. \cite{ni2013oldest,O'Leary08022013}), so data availability at the genus-level in living mammals should be our primary concern when building phylogenies of living and fossil taxa.

When few species have available anatomical characters, the ideal scenario is for them to be evenly distributed (as measured by phylogenetic overdispersion) to maximize the possibilities of a fossil branching in the right clade.
The second best scenario is that species with available anatomical characters are randomly distributed across the phylogenies. 
Here we expect no special bias in the placement of fossils \cite{GuillermeCooper}, it is therefore encouraging that for most orders, species with available anatomical characters were randomly distributed across the phylogeny.
The worst case scenario for fossil placement is that species with available anatomical characters are phylogenetically clustered. 
Then we expect two major biases to occur: first, fossils will not be able to branch within a clade containing no data, and second, fossils will have higher probability of branching within the most sampled clade by chance. 
Our results suggest that this may be problematic at the genus-level in Carnivora, Chiroptera and Soricomorpha. 
For example, a Carnivora fossil will be unable to be placed in the Herpestidae clade% rev 1.2.7
, and will have more chance to randomly branch within Canidae or Ursidae (Figure \ref{Figure_example_coverage}B).

Despite the absence of good cladistic data coverage for living mammals, %TG: I think leaving cladistic here is OK no?
 the Total Evidence method still seems to be the most promising way of combining living and fossil data for macroevolutionary analyses. 
Following the recommendations in \cite{GuillermeCooper}, we need to code anatomical characters for as many living species possible. 
Fortunately, data for living mammals is usually readily available in natural history collections, therefore, we propose that an increased effort be put into coding anatomical characters from living species, possibly by engaging in collaborative data collection projects.
Such an effort would be valuable not only to phylogeneticists, but also to any researcher focusing understanding macroevolutionary patterns and processes.

%Biology letters various stuff
\section{Ethics statement}
N/A
\section{Data accessibility statement}
All data and code are available on GitHub (\url{https://github.com/TGuillerme/Missing_living_mammals}).
\section{Authors' Contributions}
T.G. and N.C designed the study. T.G. analysed the data. T.G. and N.C. wrote the the manuscript.
\section{Competing Interests}
We have no competing interests.
\section{Acknowledgments}
We thank David Bapst, Graeme Lloyd, Nick Matzke and April Wright.
Thanks to one anonymous reviewer, Graham Slater, Peter Wagner and Graeme Lloyd for their useful comments.
%TG: That's two times Graeme though...
\section{Funding statement}
This work was funded by a European Commission CORDIS Seventh Framework Programme (FP7) Marie Curie CIG grant (proposal number: 321696).

\section{References}

\bibliographystyle{vancouver}
\bibliography{References}

%END
\end{document}
