\documentclass[12pt,letterpaper]{article}

%Packages
\usepackage{pdflscape}
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\usepackage{color}
\usepackage[usenames,dvipsnames]{xcolor}
\pagenumbering{arabic}

%Pagination style and stuff
%\linespread{2} 

\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}

\setlength\parindent{0pt}

\begin{document}

\textbf{RE: Decision on Manuscript ID RSBL-2015-1003}\\
\bigskip
Dear Surayya Johar,\\
\bigskip
We are very happy on the manuscript acceptance decision.
We have corrected the minor issues raised by referees 2 and 4 as suggested.
We also took on board the few minor comments from referees 1 and 3 and respond to their points below.

To improve the clarity of this document, we dealt with each comment in the order they appeared in your ``Decision on Manuscript'' email and we have included the reviewers comments in blue.



\section{Referee 2}

\begin{enumerate}
\item{\textcolor{blue}{line 34: spelling: ccombines = combines}}
We fixed this typo.

\item{\textcolor{blue}{lines 44 - 52. I'd suggest rewording this a little bit as lines 48 - 52 are a little awkward. May I suggest the following:
"We might expect the total evidence method to perform poorly when there is little overlap between coded anatomical characters in living and fossil taxa. This is because fossil taxa cannot be correctly placed within a clade of living species if none of its members have been coded for morphological characters. Furthermore, simulations show that fossils are more likely to be placed in clades for which more characters have been coded, regardless of whether this is the correct clade [8]."}}
We replaced the section with Referee 2's suggestion.

\item{\textcolor{blue}{line 152: ... a carnivoran fossil is unlikely to be placed in Herpestidae because ....}}
We fixed this typo.

\item{\textcolor{blue}{line 159: I'd replace "It is worth noting" with "We acknowledge"}}
We changed the start of the sentence as suggested.

\item{\textcolor{blue}{line 161: word order: provided easily accessible matrices}}
We changed the word order.

\end{enumerate}

\section{Referee 1}

\begin{enumerate}
\item{\textcolor{blue}{The authors haven’t addressed Pete Wagner’s important comments about the use of the term “cladistics.” Morphological character data are not inherently cladistics, although they can be used for cladistics analysis, as well as Bayesian and likelihood approaches. They also haven’t done anything to clarify that the relevant character set for aligning fossil and modern data is skeletal characters. Many anatomical characters do nothing at all for the incorporation of fossil taxa.}}
We emphasised the importance of skeletal characters in the introduction as follows:
``A downside of this method is that it requires molecular data for living taxa and discrete morphological/anatomical data shared between both living and fossil taxa (i.e. hard tissue characters such as skeletal ones).''
lines @@@.

\item{\textcolor{blue}{I’m also concerned that my original comments about the biases in character sets included still aren’t adequately addressed; they now acknowledge that they didn’t include any character sets that weren’t available in print, but that doesn’t address what kind of difference it would make to their results. I’m concerned it could be a really important bias; morphological data was much more commonly published for extant taxa in older phylogenetic analyses, and those are less likely to be available digitally. You might be substantially underestimating the availability of these data, and if TE phylogenies aren’t including them, I’m rather disappointed in the folks constructing them. }}

We agree with this comment (and Referee 3 comment 2) and modified the paragraph discussing this caveat as follow:
``Matrices containing anatomical characters where much more common before the advent of molecular phylogenetics and therefore are also more likely to be unavailable in a reusable format.
This might include some bias in Total Evidence analyses.
Nonetheless, these matrices are also likely to differ from more recent ones in terms of their underlying definition of homology and their coding practices (see [19]).
Additionally, many recent morphological matrices reuse living taxa from previous matrices (see ESM1).''
lines @@@.
%TG: Here's how I went with the section, basically, I acknowledge referee 1's points but add Peters ones just after:
%TG: - Yes, we specifically ignored old matrices because definition of accesible data (see above)
%TG: - Yes, we do agree it could include some bias
%TG: - But these matrices are probably not so good anyway (Peter's comment)
%TG: - And anyway it doesn't solve the problem because the living taxa used in these "old" matrices are reused in new ones anyway (or at least that's what I observed in primates)
%TG: Personally I doubt the problem is due to data availability but way more to the problem we underlined in the abstract: palaeontologist don't wanna waste their time with living species (and I don't specially blame them).


\item{\textcolor{blue}{I don’t agree with the assertion that the genus level should be our primary concern; more and more studies are building species-level trees with fossil taxa. I’m happy for the authors to discuss the patterns at the genus level, because it is one of the important levels, but given that genera are an artificial line in the taxonomic sand, it’s certainly not the primary one, and I’d hate to see this statement sail unopposed into the literature. I’d like to see it moderated a bit in the final version of the manuscript.}}
Following Referee's 1 and 4's comments, we retracted this statement.

\end{enumerate}

\section{Referee 3}

\begin{enumerate}
\item{\textcolor{blue}{The new title is an improvement.  However, I think that it could be altered still further: would something like: "Assessment of available anatomical characters for linking living mammals to fossil taxa in phylogenetic analyses"?  That is the main thrust of the paper, after all, and it also avoids having someone simply think: "Who cares? I have molecules" and moving on to the next article.}}
We changed the title as suggested by Referee 3.

\item{\textcolor{blue}{Lines 159 - 160: "It is worth noting, however, that our analysis did not include all the matrices containing anatomical characters ever published."
One thing to note is that a lot of the older analyses, which tended to be morphological, were done in the early days of modern phylogenetics, and those are the matrices that are most apt to not have pdf or html versions.  So, there might be a bias against the "pioneering" studies.  However, many of those studies will be a quarter of a century old, and the homology concepts and (especially!) coding protocols being used back then almost certainly differ from now.  Also, a lot of those matrices were "cleaned" to remove homoplasy: this was not attempted fraud, but due to the fact that computer runs that are done in hours today took weeks and weeks back then, and actually storing all of the trees that might come out of "messy" matrices was beyond the old floppy drives.  That is one reason why I rarely use matrices published before the early 1990's in my own meta-analyses.  So, if those matrices do exist and can be computerized, then I think that they should be viewed as "starting points" for coding taxa: they almost certainly will need updating.}}

% TO DO (see Referee 1 comment 2)

\end{enumerate}

\section{Referee 4}

\begin{enumerate}

\item{\textcolor{blue}{In particular I agree with Referee 1 regarding the continued use of "cladistic" and the genus-level issue (many palaeontologists are now focused on the species-level and certainly many mammalian genera are not monotypic - this is more of an issue with dinosaur workers), and Referee 3 (Pete Wagner's) title suggestion is - I think - more likely to attract readers.}}
We dealt with these suggestions (see respectively Referee 1 comment 1, Referee 1 comment 3 and Referee 3 comment 1).


\item{\textcolor{blue}{on L174 ("but also to any researcher focusing understanding macroevolutionary patterns and processes"), this seems to be missing an "on".}}
We fixed this typo.
\end{enumerate}



We hope we have responded to all these comments appropriately. Please let us know if you require any further information,\\
\bigskip


Thomas Guillerme (t.guillerme@imperial.ac.uk),\\
Natalie Cooper

\end{document}