%Authors guidlines: http://royalsocietypublishing.org/instructions-authors
% 2500 words max (includes the title page, abstract, references, acknowledgements and figure/table legends)
% current version is around 3700. I think a big cut down can be done on the references.
% We allow a maximum of 4 displays, only 2 of which can be figures.

\documentclass[12pt,letterpaper]{article}
\usepackage{natbib}

%Packages
\usepackage{pdflscape}
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
%\usepackage{natbib}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\usepackage{longtable}
\pagenumbering{arabic}


%Pagination style and stuff % NC: Note that these are all syst biol specific.
\linespread{2}
\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}
%\bibpunct{(}{)}{;}{a}{}{,}

%---------------------------------------------
%
%       START
%
%---------------------------------------------

\begin{document}

%Running head
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: Morphological data availability in living mammals

\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Morphological data is lacking for living mammals}
%Or: Missing cladistic data in living mammals - but I prefer the first one emphasizing on the availability of it. The data is not missing per se, it's just not coded!
% NC: sometimes the better title is one that answers the question rather than giving the topic. e.g., "Morphological data is lacking for living species" or similar

%Key words: Total Evidence method, data structure, phylogenetic, living, fossil, topology
\bigskip

\noindent {\normalsize \sc Thomas Guillerme$^1$$^,$$^2$$^,$$^*$ and Natalie Cooper$^1$$^,$$^2$$^,$$^3$}\\
\noindent {\small \it 
$^1$School of Natural Sciences, Trinity College Dublin, Dublin 2, Ireland.\\
$^2$Trinity Centre for Biodiversity Research, Trinity College Dublin, Dublin 2, Ireland.\\
$^3$Department of Life Sciences, Natural History Museum, Cromwell Road, London, SW7 5BD, UK.}\\
\end{center}
\medskip
\noindent{*\bf Corresponding author.} \textit{Zoology Building, Trinity College Dublin, Dublin 2, Ireland; E-mail: guillert@tcd.ie; Fax: +353 1 6778094; Tel: +353 1 896 2571.}\\ % NC: Might need to reconsider the email here as it won't be your email anymore when you leave. Or do you get to keep it forever? 
\vspace{1in}

%Line numbering
\modulolinenumbers[1]
\linenumbers

%---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------
% NC: I've commented this out so I don't have to keep scrolling past in in the PDF
%\newpage
%\begin{abstract} % 200 words, all keywords written in abstract if possible.
%Studying changes in global biodiversity through time and space is essential.
%For that we need methods to combine both palaeontological and neontological data.
%One promising method, the Total Evidence method, allows such thing but needs a lot of data.
%Especially cladistic data from living taxa to allow the fossil taxa to accurately branch in the trees.
%Despite two centuries of morphological studies on living taxa, scientists using and generating such data mainly focus on palaeontological data.
%Therefore, even in well known groups such as mammal, there is a huge gap in our knowledge of cladistic data for living mammals.
%In this study, using phylogenetic community structure methods, we quantify the availability of data in each mammalian order.
%And maybe at the end of the paper we propose some discussion on how to improve all that (go in museums!).
%\end{abstract}

%\noindent ()\\

%\vspace{1.5in}



%---------------------------------------------
%
%       INTRODUCTION
%
%---------------------------------------------

%NC: I've changed cladistic to morphological throughout but you might want to change it back...

\newpage 
\section{Introduction}
There is an increasing consensus among evolutionary biologists that studying both living and fossil taxa is essential for fully understanding macroevolutionary patterns and processes \citep{jacksonwhat2006,quentaldiversity2010,dietlconservation2011,slaterunifying2013,fritzdiversity2013,Wood01032013}.
For example, including both living and fossil taxa in evolutionary studies can improve the accuracy of timing diversification events \citep[e.g.][]{ronquista2012}, our understanding of relationships among lineages \citep[e.g.][]{beckancient2014}, and our ability to infer biogeographical patterns through time \citep[e.g.][]{Meseguer01032015}.
To perform such analyses it is necessary to combine living and fossil taxa in phylogenetic trees.
One increasingly popular method, the Total Evidence method \citep{eernissetaxonomic1993,ronquista2012}, combines molecular data from living taxa and morphological data from both living and fossil taxa in a supermatrix \citep[e.g.][]{pyrondivergence2011,ronquista2012,schragocombining2013,slaterunifying2013,beckancient2014,Meseguer01032015}, producing a phylogeny with living and fossil taxa at the tips. 
These phylogenies can be dated using methods such as tip-dating \citep{ronquista2012,Drummond01082012,Wood01032013} and incorporated into macroevolutionary studies \citep[e.g.][]{ronquista2012,Wood01032013,slaterphylogenetic2013}.

A downside of the Total Evidence method is that it requires a lot of data.
One must collect molecular data for living taxa and morphological data for both living and fossil taxa; two types of data that require fairly different technical skills (e.g. molecular sequencing \textit{vs.} anatomical description).
Additionally, large chunks of this data can be difficult, or even impossible, to collect for every taxon present in the analysis.
For example, fossils very rarely have molecular data and incomplete fossil preservation may restrict the amount of morphological data available \citep{sansomfossilization2013}. % NC: Worth metioning fossil preservation here briefly as it is also an issue?
Additionally, since the molecular phylogenetics revolution, it has become less common to collect morphological characters for living taxa when molecular data is available (e.g. in \cite{slaterphylogenetic2013}, only 13\% of the 169 living taxa have coded morphological data).
Unfortunately this missing data can lead to errors in phylogenetic inference; in fact, simulations show that the ability of the Total Evidence method to recover the correct phylogenetic topology decreases when there is a low overlap between morphological data in the living and fossil taxa \citep{GuillermeCooper}, regardless the overall amount of morphological data available for the fossils (or the amount of molecular data available for the living species).
The effect of missing data on topology is greatest when living taxa have little morphological data.
This is because (1) a fossil cannot branch in the correct clade if there is no overlapping morphological data in the clade; and (2) a fossil has a higher probability of branching within a clade with more morphological data available for living taxa, regardless of whether this is the correct clade or not \citep{GuillermeCooper}. 
%This can lead to serious topological errors (see Figure ~\ref{Figure_missing_data_problem}). 

The issues above highlight that it is crucial to have sufficient morphological data for living taxa in a clade before using a Total Evidence approach.
However, it is unclear how much morphological data for living taxa is actually available (i.e. already coded from living taxa specimens and deposited in phylogenetic matrices accessible online), and how this data is distributed.
Intuitively, most people assume this kind of data has already been collected, but empirical data suggest otherwise (e.g. in \cite{ronquista2012,slaterphylogenetic2013,beckancient2014}).
To investigate this further, we assess the amount of available morphological data for living mammals to determine whether sufficient data exists to build reliable Total Evidence phylogenies in this group.
We collected cladistic data from @256 phylogenetic matrices available online and measured the proportion of cladistic data available for each mammalian order.
%Additionally, if the available data is randomly distributed across clades, the topological errors should be less extreme than if the data is biased towards particulars groups.
Additionally, because of the two topological caveats mentioned above, we determined whether the available cladistic data was phylogenetically overdispersed or clustered in the mammalian orders where data was missing. 
We find that available morphological data for living mammals is scarce but generally randomly distributed and therefore suggest that an effort be made in collecting and sharing this type of data to improve Total Evidence tip-dated phylogenies.


%---------------------------------------------
%
%       METHODS
%
%---------------------------------------------
 
%\newpage

\section{Materials and Methods}
\subsection{Data collection and standardisation}
We downloaded all cladistic matrices containing any living or/and fossil mammal taxa from three major public databases: Morphobank (\texttt{http://www.morphobank.org/}) \citep{morphobank}, Graeme Lloyd's website (\texttt{http://graemetlloyd.com/}) and Ross Mounce's GitHub repository (\texttt{https://github.com/rossmounce}).
% NC: give link to actual repo and actual bit of graemes website these things are found on?
We also performed a systematic Google Scholar search for matrices that were not uploaded to these databases (see see Supplementary Materials for a detailed description of the search procedure).
%We downloaded the additional cladistic matrices from the 20 first search results matching with our selected keywords and with any of the 35 taxonomic levels (mammals Orders, Infraclasses and Class; see supplementary materials for detailed description of the procedure). % NC: Can possibly delete this and stick in supplement
In total, we downloaded 256 % @@@ Number of matrices
matrices containing 9411 % @@@ Number of living OTUs
operational taxonomic units (OTUs). % link
% NC: Might need to explain what you mean by OTUs, I'm assuming you mean genera or species etc.
We refer to OTUs instead of species because?
% NC: do you mean 9411 different OTUs or total OTUs? Might be good to have a number of unique OTUs as well.

% NC; probably need some comment to say this is collecting all the data!

%We transformed all the matrices to be in nexus format.
We standardised the taxonomy of our OTUs by fixing invalid binomial inputs to match with standard taxonomic nomenclature (e.g., \textit{H. sapiens} was transformed to \textit{Homo sapiens}).
We designated as ``living'' all the OTUs that where either present in the phylogeny of Fritz \textit{et al}. \citep{FritzTree} or the taxonomy of Wilson \& Reeder \citep{wilson2005mammal}, and designated as ``fossil'' all the OTUs that were present in the Paleobiology database (\texttt{https://paleodb.org} %NC: Check website
For OTUs that did not appear in these three sources we first decomposed the name (i.e. \textit{Homo$\_$sapiens} became \textit{Homo} and \textit{sapiens}) and tried to match the first element with a higher taxonomic level (Family, Genus etc.).
Any OTUs that still had no matches in the sources above were designated as non-applicable (NA; see Supplementary Material for more details).

%\subsection{Data availability analysis}
%\subsubsection{Number of characters threshold}
% NC: I'm not sure why you did this. You might need to explain more - let's discuss. Otherwise it looks like we are deliberately making the data coverage look worse by omitting data.
Next we selected only matrices containing at least 100 cladistic characters per OTU.
This arbitrary threshold was chosen to correspond with the number of characters used in \citep{GuillermeCooper} and \citep{harrisonamong-character2014}.
Additionally, this threshold avoids biases towards small matrices that are often non-informative, because they contain too few characters so there is no character overlap among matrices \citep{wagner2000}, or contain characters that are highly specific and thus non-applicable across large clades \citep{Brazeau2011} (e.g. ``antler ramifications'' is a character than is only applicable to Cervidae not all mammals).

% NC: Some summing up statement about how many matrices were actually usuable, how many OTUs, how many unique OTUs etc.

\subsection{Data availability and distribution}
To assess the availability of cladistic data for each mammalian Order, we calculated the percentage of OTUs with cladistic data at three different taxonomic levels: Family, Genus and species.
We used these different taxonomic levels because some clades are well covered at the Family or Genus level, but poorly covered at the species level.
We consider Orders with \textless 25\% of living taxa with cladistic data as having poor data coverage, and Orders with up \textmore 75\% of living taxa with cladistic data as having good data coverage. 

For Orders with \textless 100\% cladistic data coverage at any taxonomic level, we investigated whether the available cladistic data was (i) randomly distributed, (ii) over-dispersed or (iii) clustered with respect to phylogeny, using the Net Relatedness Index (NRI) metric \citep{webb2002phylogenies} in the \texttt{picante} R package \citep{picante}. % NC: Need to cite R too
NRI is a metric from community phylogenetics that quantifies the distribution of the observed data compared to a model where all the data is sampled for each OTU. 
%NC: NO! Not true unless you did something special. It assumes you have the same number of data points, but they are distributed at random. Then compares how different the observed is from random. Needs more work here. 
Negative NRI values show more over-dispersed than expected by the null model (random) and positive values are showing more clustered data \citep{webb2002phylogenies}.
We choose NRI metric because is has been shown to be slightly less sensitive to the structure of the phylogeny (i.e. branch length and topology) \citep{NRI,journal.pone.0004390} but we also calculated two other common phylogenetic structure indices: Faith's Phylogenetic Distance (PD) \citep{Faith19921} and the Nearest Taxon Index (NTI) \citep{webb2002phylogenies}.
% NC: Again No. NRI is actually skewed due to branch length distributions - see my 2008 proc b paper. Maybe worth saying it's been used in other studies then you can cite my ecology letters paper from 2012. Yay citations
Both metrics are available in the Supplementary results.
% NC: Both metrics? What other metric :) 

% NC: OK needs a bit of work, a bit more clarification of what is going on here. We can chat tomorrow.
%---------------------------------------------
%
%       RESULTS
%
%---------------------------------------------

% NC: Need to make a decision of Genus vs genus, Order vs order etc. I'd probably go for small letters actually. Do this throughout
% NC: Note Figure and Table always have capital letters. For figure 2A or 2B don't put - (not 2-B for example) 

\section{Results}

%\subsection{Data availability}
We extracted 1422 living mammal OTUs from the 256 matrices with a minimum of 6 and a maximum of 4541 coded cladistic characters per OTU.
After removing all the matrices with less than 100 cladistic characters, the number of extracted living mammals OTUs was down to 815.
% NC: Ah sorry I've asked you to add this in above. I reckon it goes up in the methods.
In the @@@ cladistic matrices extracted in our searches, 11 out of 28 mammalian orders have \textless 25\% of species with cladistic data and 24 orders have less \textless 75\% species with available cladistic data.
At the Genus level, only 3 orders have \textless 25\% of taxa with cladistic data and 16 orders have less than 75\%.
Finally, at the family level no order has \textless 25\% taxa with available cladistic data and only 5 orders have \textless 75\% (table \ref{Table_morpho_taxa_proportion}).

% NC: Results need to match the methods. So are we looking at "good" versus "poor"? That makes more sense to me. So how many have good coverage i.e. >75% and how many have poor coverage i.e. <25%. I think this will help.
% NC: Can probably also just say that there are 28 orders once, then say how many fit into each category or it's needlessly repetitive. We can write this together to save time

\renewcommand\baselinestretch{1.2}\selectfont
\begin{center}
\input{Table_morpho_taxa_proportion}
\end{center}
\renewcommand\baselinestretch{2}\selectfont

%\subsection{Available data structure}
Among the mammalian orders containing OTUs with no available cladistic data, only two orders, Carnivora and Chiroptera, are significantly phylogenetically clustered both at the species and the genus level (Table~\ref{Table_data_structure}).

\renewcommand\baselinestretch{1.2}\selectfont
\begin{center}
\input{Table_data_structure}
\end{center}
\renewcommand\baselinestretch{2}\selectfont

Two contrasting results are shown on Figure \ref{Figure_example_coverage} with randomly distributed data in Cetartiodactyla (Figure \ref{Figure_example_coverage}A) and phylogenetically clustered data in Carnivora (mainly Canidae; figure \ref{Figure_example_coverage}B).

\begin{figure}[!htbp]
\centering
    \includegraphics[width=1\textwidth]{example_coverage.pdf}
\caption{Phylogenetic distribution of available cladistic data across two mammalian orders (A: Cetartiodactyla; B: Carnivora).
Edges are colored in grey when no cladistic data is available or in red when data is available.}
\label{Figure_example_coverage}
\end{figure}
% NC: Can we change the colours, red sucks. Otherwise I like the figure a lot.

%\subsection{Data improvement strategy}


%---------------------------------------------
%
%       DISCUSSION
%
%---------------------------------------------

\section{Discussion}
Our results show that even though relations among living mammals are well known in a molecular framework \citep[e.g.][]{FritzTree,meredithimpacts2011,May-Collado-PeerJ} only little cladistic data is available.
This is primarily due to the fact that that cladistic studies are mainly focusing on fossil taxa rather than living taxa \citep[e.g.][]{O'Leary08022013,ni2013oldest}.
However, data availability varies in its structure as well as across taxonomic levels.
Higher taxonomic levels are always better sampled than lower ones (table \ref{Table_morpho_taxa_proportion}) and within these taxonomic levels, the structure of available data is mostly random apart for two orders (table \ref{Table_data_structure})b.

%\subsection{Data availability}
The amount of living taxa with no available cladistic data was surprisingly high at the species level: 24/28 orders have less than 75\% taxa with available cladistic data (and two of the 28 orders are monospecific!).
This threshold of 75\% available data is the maximum amount of possible missing data (25\%) before having a significant effect on the topology of Total Evidence trees \citep{GuillermeCooper}.
Beyond this threshold, there is a noteworthy displacement of the wildcard taxa (\textit{sensu} \citep{kearneyfragmentary2002}) as well as a important decrease in the conservation of clades \citep{GuillermeCooper}.
At the species level within most orders of mammals, it is therefore likely to see topological artefacts for the placement of fossil taxa.
However, the data availability seems to be less an issue at higher taxonomic level (i.e. the Genus and the Family level).
This point is important to consider from a practical point of view because of the slight discrepancy between the neontological and the palaeontological concept of species.
While neontological species are described on accurately measured morphology, genetic distance, spacial distribution and even behaviour \citep[e.g.][]{kellymolecular2014}, palaeontological data can be based only on morphological, spatial and temporal data \citep[e.g.][]{ni2013oldest}.
Because of this difference between neontological species (\textit{sensu} reproductive isolates) and paleontological species (\textit{sensu} morpho-species %or "morphological clusters"?
), most palaeontological studies are using the Genus level for their smallest OTU \citep[e.g.][]{ni2013oldest,O'Leary08022013}.
In this frame of palaeontological data usage, the data availability at the Genus level in living mammals should be our priority concern.

%\subsection{Available data structure}
Regardless the low level of available data, it is encouraging that for most orders (except Carnivora and Chiroptera), data is randomly distributed across the phylogeny.
%Therefore, it is likely that the addition of fossil taxa within these orders will not be biased by oversampling but will probably branch near the taxa that has the most similar available data (as expected).
There are three different scenarii, regarding the distribution of the available data, for predicting how fossils can branch in relation to living taxa.
When only few data is available, the ideal scenario will be that it is over-dispersed (i.e. that there is data in at least every sub-clade) in order to maximize the possibilities of the fossil to branch in the right clade.
In the second scenario, when the data is randomly distributed we expected no special bias in the placement of the fossil \citep{GuillermeCooper}.
However, in the third scenario, when the data is clustered we expect two major biases to occur: (1) first, the fossil will not be able to branch within a clade containing no data (e.g. Herpestidae in Carnivora; figure \ref{Figure_missing_data_problem} and \ref{Figure_example_coverage}-B), and (2) second, the fossil will have a higher probability, at random, to branch within the clade containing most of the available data (e.g. a Carnivora fossil will have more chance to branch, randomly, to the Canidae clade than any other clade in Carnivora; figure \ref{Figure_example_coverage}-B).
This means that fossils with uncertain phylogentic affinities (\textit{Incertae sedis}) will have a higher probability of branching by chance within the most sampled clade.

%For example in Carnivora, their is no available data for Herpestidae, therefore any fossil Herpestidae will not be able to be correctly placed in the phylogeny (figure ~\ref{Figure_example_coverage}-B.
%On the other hand, any Carnivora fossil with uncertain phylogenetic affinities (\textit{Incertae sedis} will have a higher probability of branching by chance within the Canidae because they are oversampled within the Carnivora (figure ~\ref{Figure_example_coverage}-B.

%Caveats
In this study, we treated all cladistic matrices to be equal in a similar way that molecular matrices are: for example if a matrix A contains 100 characters for taxa X and Y, and a matrix B contains 50 characters for taxa X and Z, we assumed that both matrices can be combined in a supermatrix containing 150 characters for taxon X, 100 for taxon Y and 50 for taxon Z.
However, it is clear that cladistic data cannot be treated that way \citep{Brazeau2011}.
In fact, for the same taxon, some characters might overlap (e.g. matrix A has a character coding for the shape of a particular morphological feature and matrix B as a first character coding for the presence of this morphological feature and a second one coding for its shape; in this case, these three characters are compound characters \citep{Brazeau2011}).
However, in reasonably sized matrices ($>$ 100 characters \citep{GuillermeCooper,harrisonamong-character2014} it is more likely that a number of characters are consistently conserved among the different matrices (e.g. within primates literature, the character \textit{p7} - the size of the $4^{th}$ lower premolar paraconid - is consistent for more than 15 years \citep{ross1998phylogenetic,seiffert2003fossil,marivaux2005anthropoid,seiffert2005basal,bloch2007new,kay2008anatomy,silcox2008biogeographic,seiffert2009convergent,tabuce2009anthropoid,boyer2010astragalar,seiffert2010fossil,marivaux2013djebelemur,ni2013oldest}).
A conservative approach to avoid compound characters could be to select only the most recent matrix for each group.

%Let's code some data
Following the recommendations in \citep{GuillermeCooper}, one should code cladistic characters for a maximum of living species in order to improve the topology of Total Evidence trees containing both living and fossil taxa.
Since the data for living mammals is usually easily available in world wide vast natural history collections, we propose that an increased effort in coding morphological characters from living species should be done via engaging in collaborative data collection projects through web portals such as \textit{morphobank} \citep{morphobank}.

%Biology letters various stuff
\section{Ethics statement}
\section{Data accessibility statement}
All data is available and reproducible on GitHub.
\section{Authors' contributions statement}
Conceived and designed the experiments: TG NC. Performed the experiments: TG. Analyzed the data: TG. Wrote the paper: TG NC. % NC: Write this in the proper proceedings style. All authors gave final approval for publication.
\section{Acknowledgements}
Nick Matzke, April Wright, David Bapst and Graeme Lloyd. %Or whatever order. % NC: Alphabetical?
\section{Funding statement}
This work was funded by a European Commission CORDIS Seventh Framework Programme (FP7) Marie Curie CIG grant (proposal number: 321696).

\bibliographystyle{sysbio}
\bibliography{References}


%\section{SOM}
\newcommand{\beginsupplement}{%
    \setcounter{table}{0}
    \renewcommand{\thetable}{S\arabic{table}}%
    \setcounter{figure}{0}
    \renewcommand{\thefigure}{S\arabic{figure}}%
}
\beginsupplement
\input{Supplementary/Supplementary}


%END
\end{document}
